recipes/rmf/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mkdir build && cd build
+cmake -DCMAKE_BUILD_TYPE=Release \
+      -G Ninja \
+      -DCMAKE_PREFIX_PATH=${PREFIX} \
+      -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+      -DCMAKE_INSTALL_PYTHONDIR=${SP_DIR} \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      ..
+ninja install -j${CPU_COUNT}
```

Comment:
No need for `CPU_COUNT` with `ninja`
```suggestion
ninja install
```